build-system/common/check-package-manager.js

```diff
@@ -310,13 +310,19 @@ function runGulpChecks() {
 }
 
 function checkPythonVersion() {
-  // Python prints its version to stderr: https://bugs.python.org/issue18338
-  const pythonVersionResult = getStderr(`${pythonExecutable} --version`).trim();
+  // Python2 prints its version to stderr (fixed in Python 3.4)
+  // See: https://bugs.python.org/issue18338
+  const pythonVersionResult =
+    getStderr(`${pythonExecutable} --version`).trim() ||
+    getStdout(`${pythonExecutable} --version`).trim();
```

Comment:
Nice!

Reply:
I'm glad you're ok with this. I wasn't as proud of it (two process execs instead of one for Python3 users). I would prefer something like `getStderrStdout` that effectively acts like `2>&1`, but `child_process` doesn't make this kind of thing easy. I decided this would be ok since the eventual removal of python 2.7 would reduce this back down to one exec.